dev/src/webpack/getDefaultWebpackConfig.ts

```diff
@@ -211,6 +209,10 @@ export function getDefaultWebpackConfig(options: DefaultWebpackOptions): webpack
                     "async_hooks": false,
                     "child_process": false,
                     "fs": false,
+                    'html-to-text': false,
```

Comment:
This package is used to parse errors. I'm not sure how to really test it (because I don't see html errors often) but it's wrapped in a try/catch so I don't think that this should cause any exceptions to be thrown in a vscode.dev environment.

Reply:
Hasn't this one given us CG alerts before? Just to parse errors feels like a pretty flimsy reason to keep it...

Reply:
It's only used to parse error messages in case the error message is an html document, which seems somewhat common for Azure errors. See https://github.com/microsoft/vscode-azuretools/pull/360

Reply:
Also, I couldn't quickly find any alternatives that are browser compatible, so 🤷

Reply:
Oh the irony...can't find a browser-compatible package to parse HTML